src/QuizPage/Answer.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const StyledAnswer = styled.div`
+    background-color: #fff;
+    border: 2px solid ${props => props.borderColor};
+    margin-bottom: 20px;
+    cursor: pointer;
+    font-size: 18px;
+    font-weight: 400;
+    padding: 14px 65px;
+    box-sizing: border-box;
+    flex-basis: 50%;
+`;
+
+const htmlDecode = (input) => {
+    return input.replace(/&#039;/g, "'")
+                .replace(/&quot;/g, "\"");
+}
+
+class Answer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            color: "#0094da",
+        }
+    }
+    
+    checkAnswer = (event, answers, correct_answer) => {
```

Comment:
checking if this is the correct answer could be already done on the quiz page upon render, passing it as a boolean, then you would not need to have a lots property passed down such as correct_answer, answers or id (which btw is an index, so it is not a good name)